### ISSUE TITLE
Couldnt use --quiet to suppress the output from b2 account authorize

### ISSUE THREAD

**Issue by sppidy** [Backblaze/B2_Command_Line_Tool#1035]
Couldnt use --quiet to suppress the output from b2 account authorize
i am trying to use b2 in public server while authentication it prints out the session info in json like format where all our keys are visible

**Comment by mjurbanski-reef** [comment]
This is by choice - `--quite` suppresses only human readable information and all machine readable output is not silenceable by `--quite`. We need to improve the documentation of that.

If you really need to silence everything no matter on POSIX systems you can still use `b2 ... > /dev/null 2>&1` .

What is concerning to me in this report is the motivation to do so - what do you mean by "public server"?
`b2 account authorize` persists credential in the filesystem, so anyone with access to it can gain access to the account. This typically is easier than than accessing terminal output.

The cases that Im aware of in which silencing of `b2 account authorize` would be useful is the use in CI to prevent logs from containing auth credentials and when screen sharing/presenting.

**Comment by sppidy** [comment]
Thanks for replying. Actually i am part a group of people using a server with a queueing system where we build custom roms and sign it and i am personally using back-blaze to store my keys and retrieving the keys via script which also sign the builds and clears the locally stored keys and clears the credentials of the backblaze too.  I raised this issue because i was concerned about my auth keys as it could be seen by any one. The important part is everyone using that server can see the the current status and logs of the builds. So thats why I raised this issue . Anyway thanks for reply my query
